CHAIN_INTEGRATION.md

```diff
@@ -0,0 +1,75 @@
+# Substrate Api Sidecar chain integration guide
+
+This guide aims to help chain builders integrate their Substrate FRAME based chain with Substrate API Sidecar.
+
+## Table of contents
+
+- [Polkadot-js API type definition support](#polkadot-js-API-type-definition-support)
+- [Controller configuration](controller-configuration)
+
+## Polkadot-js API type definition support
+
+In order decode the SCALE encoded data from a substrate based node, polkadot-js needs to have a registry of type definitions. Sidecar pulls in chain type definitions from the [@polkadot/apps-config package hosted on NPM](https://www.npmjs.com/package/@polkadot/apps-config).
+
+If the chain's type definitions do not already exist in [@polkadot/apps-config](https://github.com/polkadot-js/apps/tree/master/packages/apps-config) they will need to be added via PR to polkadot-js/apps by following their [instructions for API config](https://github.com/polkadot-js/apps/tree/master/packages/apps-config#api).
+
+Before taking any other steps to integrate a chain with Sidecar, a chain's up-to-date type definitions must be included in a published version of @polkadot/apps-config.
+
+## Controller configuration
+
+Sidecar offers the ability to configure which controllers to mount. Sidecar uses a chain's spec name to determine which controller config to use, and if no config is linked to a spec name, then the [default config](/src/chains-config/defaultControllers.ts) is used.
+
+A chain builder can follow the below steps and submit a chain's controller config via PR, where it will be reviewed and merged once deemed ready by the maintainers.
```

Comment:
```suggestion
A chain builder can follow the steps below and submit a chain's controller config via PR, where it will be reviewed and merged once deemed ready by the maintainers.
```